src/main/java/seedu/address/logic/commands/SetCommand.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.logic.commands;
+
+/**
+ * Sets predicates that filter article book
+ */
+public class SetCommand {
```

Comment:
Why not "FilterCommand"

Reply:
I wanted to emphasize that using the command **sets** the filter for all future seraches.